### PR TITLE
port of iosp-docs

### PR DESCRIPTION
**Pull request by haileyajohnson**
port of #523

---
This change is [Reviewable](https://reviewable.io/reviews/unidata/netcdf-java/525)

### CODE REVIEW

File: docs/src/site/pages/netcdfJava/StructureDataArraysRef.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
```
Comment (haileyajohnson) [inline docs/src/site/pages/netcdfJava/StructureDataArraysRef.md:1]: rewriting the content of this file will come in a later PR

**Comment by lesserwhirls** [comment]
After https://github.com/Unidata/netcdf-java#636 goes in, would you mind rebasing this PR? I can ping you on slack to let you know when it's ready if that's easier. Thanks @haileyajohnson!

**Comment by lesserwhirls** [comment]
Ok, cool - one more merge or rebase and we'll be good to go with this one. Thank you @haileyajohnson!

**Review by lesserwhirls — APPROVED** [review]
Great work @haileyajohnson! 

> Reviewed 31 of 33 files at r1, 2 of 2 files at r2.
> *[Reviewable](https://reviewable.io/reviews/unidata/netcdf-java/525#-:-MX23i3n1ROv2Klqyb69:bscqhum)* status: all files reviewed, 1 unresolved discussion (waiting on @haileyajohnson)

___
*[docs/src/site/pages/netcdfJava/StructureDataArraysRef.md, line 1 at r1](https://reviewable.io/reviews/unidata/netcdf-java/525#-MWf_mlZiUqSoPpLRE80-r1-1:-MX22R6OCP8No0cTiLFa:b4c9lyr) ([raw file](https://github.com/unidata/netcdf-java/blob/de846d9bf0730d281191111cdb28ebfb90bfe43a/docs/src/site/pages/netcdfJava/StructureDataArraysRef.md#L1)):*
<details><summary><i>Previously, haileyajohnson wrote…</i></summary><blockquote>

rewriting the content of this file will come in a later PR
</blockquote></details>

:thumbsup:

**Pull request merged**